[Visual] I need it colorful

## 🐛 Visual Bug Report

**Severity:** 🟡 Minor
**URL:** `/`
**Viewport:** 1397×661
**Component:** `Sidebar`
**Selector:** `#root > div.flex.bg-background > aside.left-0.top-0 > div:nth-of-type(1).flex.items-center`
**Route:** `/`

### User Description
Add more colors to it.

### Screenshot
![Bug Screenshot](https://raw.githubusercontent.com/pranav-cholleti/ecommerce-app/main/.issueres/screenshots/visual-report-1775495712146.png)

### Console Errors
```
[ERROR] 10:30:40 pm — Failed to refresh data: rawData is not defined
[ERROR] 10:30:40 pm — Uncaught ReferenceError: Uncaught ReferenceError: rawData is not defined at http://localhost:5173/src/components/Header.tsx:20:5
[ERROR] 10:30:46 pm — Network error during sync: Failed to fetch
```

### Failed Network Requests
| URL | Method | Status | Duration |
|:----|:-------|:-------|:---------|
| `/v1/stock/sync` | POST | 0 | 1853ms |

### Environment
- **Branch:** `main`
- **Commit:** `demo-hash-12345`

---
*Generated by IssueRes Visual Reporter*

<!-- ISSUERES_META: {"source":"visual-reporter","targetComponent":"Sidebar","parentComponent":null,"cssSelector":"#root > div.flex.bg-background > aside.left-0.top-0 > div:nth-of-type(1).flex.items-center","route":"/","viewport":{"width":1397,"height":661},"consoleErrors":["Failed to refresh data: rawData is not defined","Uncaught ReferenceError: Uncaught ReferenceError: rawData is not defined at http://localhost:5173/src/components/Header.tsx:20:5","Network error during sync: Failed to fetch"],"failedRequests":[{"url":"/v1/stock/sync","method":"POST","status":0}],"environment":{"branch":"main","commitHash":"demo-hash-12345"},"severity":"minor"} -->

--- a/src/components/Sidebar.tsx
+++ b/src/components/Sidebar.tsx
@@ -3,11 +3,11 @@ import { Home, BarChart2, ShoppingBag, Users, Settings, LogOut } from 'lucide-re
 
 const Sidebar: React.FC = () => {
   const menuItems = [
-    { name: 'Dashboard', icon: <Home size={20} />, active: true },
-    { name: 'Analytics', icon: <BarChart2 size={20} />, active: false },
-    { name: 'Products', icon: <ShoppingBag size={20} />, active: false },
-    { name: 'Customers', icon: <Users size={20} />, active: false },
-    { name: 'Settings', icon: <Settings size={20} />, active: false },
+    { name: 'Dashboard', icon: <Home size={20} />, active: true, color: 'text-primary' },
+    { name: 'Analytics', icon: <BarChart2 size={20} />, active: false, color: 'text-blue-500' },
+    { name: 'Products', icon: <ShoppingBag size={20} />, active: false, color: 'text-green-500' },
+    { name: 'Customers', icon: <Users size={20} />, active: false, color: 'text-yellow-500' },
+    { name: 'Settings', icon: <Settings size={20} />, active: false, color: 'text-purple-500' },
   ];
 
   return (
@@ -25,14 +25,14 @@ const Sidebar: React.FC = () => {
         {menuItems.map((item) => (
           <button
             key={item.name}
-            className={`w-full flex items-center gap-3 px-4 py-3 rounded-xl transition-all duration-200 ${
+            className={`w-full flex items-center gap-3 px-4 py-3 rounded-xl transition-all duration-200 ${ 
               item.active 
                 ? 'bg-primary/10 text-primary border border-primary/20 shadow-lg shadow-primary/5' 
-                : 'text-white/40 hover:text-white hover:bg-white/5'
+                : `${item.color} hover:text-white hover:bg-white/5` 
             }`}
           >
             {item.icon}
-            <span className="font-medium">{item.name}</span>
+            <span className={`font-medium ${item.active ? '' : item.color}`}>{item.name}</span>
           </button>
         ))}
       </nav>

